tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/model/Label.scala

```diff
@@ -28,6 +34,31 @@ case class DisplayName(value: String)
 
 case class Color(value: String)
```

Comment:
need to add a ColorConstraint upon Color object creation IMO.

Reply:
That it should be a valid color in hexa value? Will think about it, makes sense